### DISCUSSION TITLE
Explicacion de switch

### DISCUSSION THREAD

**Discussion by rglvlncsr** [atralice/Curso.Prep.Henry#40] [Ideas]
Ejercicio 03 falta explicar tanto en el video como lo escrito el statement switch.
Como podriamos agregarlo al repo?

**Comment by atralice** [comment]
Buenas, como va?
Es buena idea agregarlo! lo pongo en el backlog.

Tambien podés hacer un PR en la teoría para explicar el statement switch si queres que esté más rápido.
Abrazo.
